R/compute_weighted_intensity_levels.R

```diff
@@ -88,12 +77,13 @@ fit_peak <- function(
   # Check input arguments
   coll <- checkmate::makeAssertCollection()
   checkmate::assert_tibble(weighted_observations, add = coll)
+  checkmate::assert_numeric(conf_levels, lower = 0, upper = 1, len = 3)
```

Comment:
This is just a stray thought, but in principle we could be very specific about conf_levels.
`assert_numeric` also takes arguments `unique = TRUE` and `sorted = TRUE` which could be used, since you 
implicitly require it in the package. However, it is also more work to think of all requirements we have on variables, so not sure if it worth implementing such explicit checks throughout. Food for thought.

Reply:
That is a good idea, thanks :)